GOVERNANCE.md

```diff
@@ -0,0 +1,130 @@
+# Release Working Group
+
+The Node.js Release Working Group (WG) maintains oversight
+over the Node.js Release and Long Term Support (LTS) teams. It
+manages the release and long term support shedule and policies
+for all Node.js releases.
+
+The WG has final authority over Releases including:
+
+* Release process and tools.
+* Content for all releases.
```

Comment:
I believe that using the standard delegation of responsibility from the CTC to the WG that is correct.  As a final measure the CTC could dissolve the WG in order to take back responsibility if the WG is not acting in accordance with the CTC's vision.